assessingsolar/_toc.yml

```diff
@@ -34,6 +34,8 @@ parts:
 #      title: Satellite Data
 #    - file: notebooks/reanalysis_data
 #      title: Reanalysis Data
+    - file: error_gallery_outputs/index.rst
```

Comment:
`error_gallery_outputs` is part of the URL for these pages.  I point this out in case you want to change it to something nicer.

Reply:
I renamed it to "gallery", but I guess it would have been more telling to name it "errors", "faults", or "quality_issues". Especially in case that other galleries are added in the future.

Reply:
I renamed it to "gallery", but I guess it would have been more telling to name it "errors", "faults", or "quality_issues". Especially in case that other galleries are added in the future.

Reply:
I renamed it to "gallery", but I guess it would have been more telling to name it "errors", "faults", or "quality_issues". Especially in case that other galleries are added in the future.